perf/istio/gracefulShutdown/templates/server.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: httpbin
+  labels:
+    app: httpbin
+spec:
+  ports:
+  - name: http
+    port: 8000
+    targetPort: 80
+  selector:
+    app: httpbin
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: httpbin
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: httpbin
+    spec:
+      containers:
+      - image: docker.io/kennethreitz/httpbin
```

Comment:
Please use fortio here if it supports those options.

Reply:
I was initially using fortio, but the delay for fortio is maxed out at 1.5s. httpbin also has a limit, but it is 10s. Having a longer delay may expose more issues, but 1.5s is also pretty long, so we could switch to that if you want.

Reply:
Ok httpbin fine then.